frontend/src/lib/components/candidates/CandidateDetailsCard.svelte

```diff
@@ -17,30 +17,30 @@
   };
 </script>
 
-<section class="mt-4">
+<section>
   <!-- TODO: replace the following div tag with the EntityCard component -->
-  <div class="mb-4 ml-4">
-    <h1 class="text-h1">
+  <header class="p-lg">
```

Comment:
Do we need to define a header? Does it make a difference for the screen readers, or is it just for grouping the candidate details?

Reply:
Screen readers will treat it as a `section` but it makes sense here semantically (see [MSDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)), as I'm also changing the wrapper element to `article` because it contains the whole candidate.